tex/chSpecification.tex

```diff
@@ -1035,15 +1035,15 @@ \section{Strong induction via inductive specs}
 obtains the same effect of \C{move: {-2}n (leqnn n)}.
```

Comment:
thanks for the fix! I think this paragraph also needs to be updated.

Reply:
Haha, I've spent a moment pondering if this is a "separate enough" part of text to use a generic `n` name
I've never before realized that writing math/sci/tech text actually requires such non-trivial style decisions :D